Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2627,6 +2627,7 @@ DigitalCredentialsEnabled:
       default: false
     WebCore:
       default: false
+  sharedPreferenceForWebProcess: true
 
```

Comment:
Would you mind adding the following line to include "DigitalCredentialsEnabled" as part of the "richJavascript" feature SPI please.
"richJavaScript: true"